### ISSUE TITLE
My Replies in conversations appears empty

### ISSUE THREAD

**Issue by weweje** [protz/thunderbird-conversations#1098]
Hi and thank you for this beautifull piece of code that gave my TB a 2k look&feel.
However my replies in conversations are missing: they appears as a blank email with no content, even if I open them in the classic reader.
SEE [HERE](https://i.imgur.com/YHGAEnB.png) for a screenshot.
Thank you for enventually investing that matter.
yours.
![reply bug](https://cloud.githubusercontent.com/assets/2323353/15894636/35ae7cd0-2d87-11e6-9e2a-217ccfe83530.PNG)

**Comment by reinsmedinga** [comment]
I have the exact same problems with a Dutch version of Thunderbird in combination with the conversations add-on.  I am using TH 45.1.1 at the moment. Are the perhaps any special options in TB-conversions I should use to prevent these empty messages...

**Comment by reinsmedinga** [comment]
I just deleted the enigmail add-on and it looks like all sent-messages seems te be visible again. Can enigmail be the problem?

**Comment by weweje** [comment]
I do also have enigmail so it may be related, but had rather uninstalled thunderbird-conversations which moreover made my whole (massive ) inbox laggy.

reinsmedinga wrote on 19/07/2016 15:46:

**Comment by kynan** [comment]
Also suffering from the same issue with thunderbird-conversations + enigmail. What's even more bizarre is that the preview of the reply in the conversation overview *is* actually shown.

**Comment by timn** [comment]
Same here, versions the same as mentioned by kynan. This is on Fedora 23. Disabling enigmail is a workaround.

**Comment by protz** [comment]
I'd love to get more information -- do you have something meaningful in the error console? Does this happen every time or just for certain emails?

Thanks,

Jonathan

**Comment by kynan** [comment]
This happens for every email consistently. What's also surprising:

1. The preview in the conversation overview *is* shown
2. The email is also empty when viewing using the classic reader
3. The email is also empty when composing a reply

I have disabled Enigmail to work around the issue but can re-enable it and see if I get any console output.

**Comment by protz** [comment]
Yes, that would be helpful. Thanks!

**Comment by seb45tian** [comment]
Same issue here. TB 45.2.0 + Enigmail 1.9.6.1 + Conversations 2.11.4. S/MIME signed messages are not shown (PGP signed messages are shown correctly). Preview of contents IS shown. Also when forwarding the message the contents are shown correctly. Classic view does not work  as well. Added some screenshots below.

Unfortunately there is no error-console output I can show you. There is none. Is there any other way I could narrow down the problem?Disabling Enigmail helps for the moment.

 Thanks for your hard work!

![conversations_1](https://cloud.githubusercontent.com/assets/11886615/22201577/7912c2b0-e164-11e6-9d31-fdcb0118cee1.png)

![conversations_2](https://cloud.githubusercontent.com/assets/11886615/22201608/9087a938-e164-11e6-9d11-1d41ce730991.png)

**Comment by kynan** [comment]
Nothing useful in the console unfortunately, the only thing I got was:
* Warning: unreachable code after return statement
Source File: chrome://conversations/content/js/jquery.tokeninput.js
Line: 470
Timestamp: 23/01/17 18:58:33
* Warning: Use of getPreventDefault() is deprecated.  Use defaultPrevented instead.
Source File: chrome://conversations/content/js/jquery-1.6.4.min.js
Line: 3

**Comment by sebastianst** [comment]
I had the same problem. Now it got worse: my replies are missing from all conversation threads entirely. Anyone else experiencing this? Should I open a new issue?
TB 52.2.1 Enigmail 1.9.8.1 Conversations 2.11.4

**Comment by shirosaki** [comment]
I've confirmed the issue.
TB can load multiple messages in a thread in parallel.
But Enigmail seems not to allow parallel verification/decryption.
https://sourceforge.net/p/enigmail/source/ci/enigmail-1.9.8.3/tree/package/mimeDecrypt.jsm#l86
Increasing `MAX_NUM_PROC` may work.
Or loading singed/encrypted messages in sequence is another solution.

**Comment by Vitlik** [comment]
Increased  `MAX_NUM_PROC` to 20 and to opened a single message (no long conversation).
Signed messages still empty and encrypted message stay encrypted even though labeled "decrypted" on top.
So, it did solve the issue...

**Comment by Standard8** [comment]
A lot has changed in the last couple of years, in Enigmail, Thunderbird and Converations.

If this is still an issue with the latest version of each, please feel free to file a new issue. Please include full system details and any other add-ons you have, and anything showing on the error console.
